<?xml version="1.0" encoding="utf-8"?>
<launch>
    <node pkg="sawyer_control" type="arm_joint_action_server.py" name="arm_servers"/>
    <node pkg="sawyer_control" type="robot_state_allinone_server.py" name="robot_state_allinone_server" output="screen"/>
    <node pkg="sawyer_control" type="robot_state_server.py" name="robot_state_server" />
    <node pkg="sawyer_control" type="jacobian_server.py" name="jacobians" />
    <node pkg="sawyer_control" type="image_server.py" name="images" />
    <node pkg="sawyer_control" type="angle_action_server.py" name="angle_action" />
    <node pkg="sawyer_control" type="ik_server.py" name="ik_server" />
<!--     <node pkg="sawyer_control" type="object_pose_publisher.py" name="objects_pose_publisher" /> -->
<!--     <node pkg="sawyer_control" type="workspace_state_server.py" name="workspace_state_server" /> -->
</launch>
